src/Raven.Client/Documents/Session/IAsyncSessionDocumentTimeSeries.cs

```diff
@@ -15,66 +15,82 @@ namespace Raven.Client.Documents.Session
     /// <summary>
     ///     Advanced async TimeSeries session operations
     /// </summary>
-    public interface IAsyncSessionDocumentTimeSeries : 
-        ISessionDocumentAppendTimeSeriesBase, 
+    public interface IAsyncSessionDocumentTimeSeries :
+        ISessionDocumentAppendTimeSeriesBase,
         ISessionDocumentDeleteTimeSeriesBase,
         IAsyncTimeSeriesStreamingBase<TimeSeriesEntry>
     {
+        /// <summary>
+        /// Get Time-Series entries asynchronously.
+        /// </summary>
+        /// <param name="from">Date time to start collecting from (included). By default: DateTime.MinValue.</param>
+        /// <param name="to">Date time to stop collecting from (included). By default: DateTime.MaxValue.</param>
+        /// <param name="start">Number of time series entries that should be skipped. By default: 0.</param>
+        /// <param name="pageSize">Maximum number of time series entries that will be retrieved. By default: int.MaxValue.</param>
+        /// <param name="token"></param>
         Task<TimeSeriesEntry[]> GetAsync(DateTime? from = null, DateTime? to = null, int start = 0, int pageSize = int.MaxValue,
             CancellationToken token = default);
 
+        /// <inheritdoc cref="GetAsync"/>
+        /// <param name="includes">Action to include Tags or Documents.</param>
```

Comment:
That's nice. I wasn't aware that you can use `<inheritdoc ... ` + extend it providing additional parameters

Reply:
Please also note that it works the other way around. If you document a method which has the all of the parameters and then you apply `<inheritdoc ...` to a method which has fewer parameters then it will work correctly as well. That's should be the best approach in general (not necessary here).